src/AccessibilityInsights.SharedUx/Settings/ConfigurationManager.cs

```diff
@@ -164,18 +164,9 @@ public void SaveEventsConfiguration()
         /// <returns></returns>
         public static ConfigurationManager GetDefaultInstance(FixedConfigSettingsProvider provider = null)
         {
-            if(sDefaultInstance == null)
+            if (sDefaultInstance == null && provider != null)
```

Comment:
Was `provider == null` the only situation we expect an exception here? I feel a little nervous removing the try...catch. Though I see that most of the work under the ConfigurationManager constructor is try/caught already.